README.md

```diff
@@ -1,132 +1,212 @@
-is `make static`, which will download v8 source at a specific version and build
-it, and statically link plv8 to it.  PGXN install will use the former, while
-you can do the latter manually if you have not installed v8 yet.
-
-Once you have built and installed plv8, create the language via:
-
-```
-  $ psql -c 'CREATE EXTENSION plv8'
-  $ psql -c 'CREATE EXTENSION plls'
-  $ psql -c 'CREATE EXTENSION plcoffee'
-```
-
-in 9.1 or later, or in the prior versions:
-
-```
-  $ psql -f plv8.sql
-```
-
-to create database objects.
-
-As of 2.0.0, there is a function to determine which version of plv8 you have
-installed:
-
-```
-  $ psql -c 'SELECT plv8_version()'
-```
-
-In mingw64, you may have difficulty in building plv8.  If so, try to make
-the following changes in Makefile.
-
-```
-  CUSTOM_CC = gcc
-  SHLIB_LINK := $(SHLIB_LINK) -lv8 -Wl,-Bstatic -lstdc++ -Wl,-Bdynamic -lm
-```
-
-For more detail, please refer to https://github.com/plv8/plv8/issues/29
-
-TEST
-----
-
-plv8 supports installcheck test.  Make sure set custom_variable_classes = 'plv8'
-in your postgresql.conf (before 9.2) and run
-
-```
-  $ make installcheck
-```
-
-EXAMPLE (JAVASCRIPT)
---------------------
-
-```
-  CREATE OR REPLACE FUNCTION plv8_test(keys text[], vals text[])
-  RETURNS text AS $$
-    var o = {};
-    for(var i=0; i<keys.length; i++){
-      o[keys[i]] = vals[i];
-    }
-    return JSON.stringify(o);
-  $$ LANGUAGE plv8 IMMUTABLE STRICT;
-
-  SELECT plv8_test(ARRAY['name', 'age'], ARRAY['Tom', '29']);
-           plv8_test
-  ---------------------------
-   {"name":"Tom","age":"29"}
-  (1 row)
-```
-
-EXAMPLE (COFFEESCRIPT)
-----------------------
-
-```
-  CREATE OR REPLACE FUNCTION plcoffee_test(keys text[], vals text[])
-  RETURNS text AS $$
-    return JSON.stringify(keys.reduce(((o, key, idx) ->
-      o[key] = vals[idx]; return o), {}), {})
-  $$ LANGUAGE plcoffee IMMUTABLE STRICT;
-
-  SELECT plcoffee_test(ARRAY['name', 'age'], ARRAY['Tom', '29']);
-         plcoffee_test
-  ---------------------------
-   {"name":"Tom","age":"29"}
-  (1 row)
-```
-
-EXAMPLE (LIVESCRIPT)
---------------------
-
-```
-  CREATE OR REPLACE FUNCTION plls_test(keys text[], vals text[])
-  RETURNS text AS $$
-    return JSON.stringify { [key, vals[idx]] for key, idx in keys }
-  $$ LANGUAGE plls IMMUTABLE STRICT;
-
-  SELECT plls_test(ARRAY['name', 'age'], ARRAY['Tom', '29']);
-           plls_test
-  ---------------------------
-   {"name":"Tom","age":"29"}
-  (1 row)
-```
-
-NOTES
------
-plv8 is hosted on github
-https://github.com/plv8/plv8
-
-and distributed by PGXN.  For more detail, see
-http://pgxn.org/dist/plv8/doc/plv8.html
+- clang++
+
+Also all tools that PostgreSQL and V8 require to be built are required if you
+are building those from source.
+
+# Installing plv8
+
+## Build from source:
+```shell
+$ git clone git@github.com:plv8/plv8.git
+$ cd ./plv8
+$ git checkout tags/v1.5.4 -b 1.5.4
```

Comment:
I would rather not get people checking out branches unless they have very special requirements.  for instance, `1.4` is maintained strictly for Debian compatibility, as they work their way to a modern `v8`.  `1.5` is maintained for legacy at this point as well.

great pains are made to not break master, where development on modern `v8` is being done, and more modern OS's will either ship with a newer `libv8`, or will require a more recent version of `v8` in order to compile at all without hacks.